packages/contracts/deploy/90_erc20-voting-factory.ts

```diff
@@ -0,0 +1,42 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+import {getContractAddress} from './helpers';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts, ethers} = hre;
+  const {deploy} = deployments;
+
+  const {deployer} = await getNamedAccounts();
+
+  const tokenFactoryAddress = await getContractAddress('TokenFactory', hre);
+
+  const ret = await deploy('ERC20VotingFactory', {
+    from: deployer,
+    args: [tokenFactoryAddress],
+    log: true,
+  });
+
+  // Create Repo and register on Aragon Plugin Registry
+  const factoryAddress: string = ret.receipt?.contractAddress || '';
+
+  if (factoryAddress !== '') {
```

Comment:
What's the use case here? How would be not want to deploy something? Is redeployment parameterizable? How would versioning work here?